doc/api/training/smd_data_parallel_release_notes/smd_data_parallel_change_log.rst

```diff
@@ -0,0 +1,170 @@
+Sagemaker Distributed Data Parallel 1.2.1 Release Notes
+=======================================================
+
+**New Features:**
+
+-  Added support for TensorFlow 2.5.0 and PyTorch 1.9.0.
+
+**Improvements**
+
+-  Improved performance on a single node.
+-  Improved performance on small clusters (2-4 nodes).
+-  Improved performance of ``Accumulator``.
```

Comment:
This line `Improved performance of ``Accumulator``` is too technical and duplicates 2-4 nodes perf improvements. We can remove it.

Reply:
removed the line